src/hatti/views/dataview.cljs

```diff
@@ -100,8 +100,9 @@
             dataviews (->> app-state :views :all
                            (map dataview-map) (remove nil?))
             dv->link (fn [{:keys [view label]}]
-                       (if (and (= view :map) no-geodata?)
-                         [:a {:class "inactive" :title "No geodata"}
+                       (if (or (and (= view :map) no-geodata?)
+                               (in? (-> app-state :views :disabled) view))
+                         [:a {:class "inactive" :title "No data"}
```

Comment:
That's OK, it's a separate case from the "No geodata" and should be handled
as such.

Eventually we'll want to translate this, but we can ignore that for now.